Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -812,7 +812,16 @@
                         .withId("note")
                         .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
                         .withContentOverflow()
-                        .withSourceValueEchoRendering());
+                        .withSourceValueEchoRendering())
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("RiskAssessmentDate")
+                        .withTitle("Risk assessment date <- change the text")
+                        .withId("LatestRiskAssessmentDate")
+                        .withDataSourceType(Utility.KendoGrid.KendoGridColumnDataSourceType.Date)
+                        .withStandardWidth(170)
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderDate(dataItem.RiskAssessmentDate))
+                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderDate(dataItem.RiskAssessmentDate)));
```

Comment:
not following your own learnings here... no need to repeat the same rendering if the UI rendering is OK for excel output